core/builtins/parser/message.py

```
@@ -308,16 +308,16 @@ async def _execute_module(msg: "Bot.MessageSession", modules, command_first_word
             else:
                 Logger.debug("Tos is disabled, check the configuration if it is not work as expected.")
 
-        none_doc = True  # 检查模块绑定的命令是否有文档
+        none_templates = True  # 检查模块绑定的命令是否有模板
         for func in module.command_list.get(msg.session_info.target_from):
-            if func.help_doc:
-                none_doc = False
-        if not none_doc:  # 如果有，送入命令解析
+            if func.command_template:
+                none_templates = False
+        if not none_templates:  # 如果有，送入命令解析
```

Ruff (0.14.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)